doc/UserGuide.md

```diff
@@ -45,7 +45,7 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > Persons can have any number of tags (including 0)
 
 Examples: 
```

Comment:
before the example, you need to let the user know of the actual format :-)
hence you need to update the format above in this user guide